README.md

```diff
@@ -13,6 +13,19 @@ bundle exec rake bootstrap
 bundle exec rake build
 ```
 
+### Common issues
+
+<details><summary>Issues installing `therubyracer` and/or `v8` dependencies</summary>
+Some combination of the following might help resolve issues with installing these dependencies:
+
+- Installing `v8` via homebrew: `brew install v8`
+- Installing the `libv8` gem using a specific version and v8 flag:
+  `gem install libv8 -v '3.16.14.19' -- --with-system-v8`
+- Assigning configuration options, as in
+  [this comment](https://gist.github.com/fernandoaleman/868b64cd60ab2d51ab24e7bf384da1ca#gistcomment-3114668).
+
+</details>
+
```

Comment:
Every change after this line is reformatting churn. The only new content is ☝️ .